clap_mangen/src/lib.rs

```diff
@@ -92,6 +92,51 @@ impl Man {
     }
 }
 
+/// Handle [`Man`] in relation to files
+impl Man {
+    /// Generate the filename of the manual page
+    #[must_use]
+    pub fn get_filename(&self) -> String {
+        format!(
+            "{}.{}",
+            self.cmd
+                .get_display_name()
+                .unwrap_or_else(|| self.cmd.get_name()),
+            self.section
+        )
+    }
+
+    /// [Renders](Man::render) the manual page and writes it to a file
+    pub fn generate_to<Dir>(&self, out_dir: Dir) -> Result<std::path::PathBuf, std::io::Error>
+    where
+        Dir: AsRef<std::path::Path>,
```

Comment:
Personally I like readability. Shortening everything to single letters does not help with readability. Especially when there is more than one generic / lifetime. In this case it's already quite clear what is expected with the variable name `out_dir` so it could be shortened.

Reply:
Except that is not the style of this code base.